lldb/include/lldb/Utility/Status.h

```diff
@@ -91,10 +136,14 @@ class Status {
 
   ~Status();
 
-  // llvm::Error support
-  explicit Status(llvm::Error error) { *this = std::move(error); }
+  /// Try not to use this in new code. Migrate APIs to llvm::Expected instead.
```

Comment:
[nit] I'd word this even stronger: "Avoid using this in new code.  Migrate APIs to use llvm::Error or llvm::Expected directly."